### PR TITLE
EZP-28688: Update enterprise tutorial to 2.3

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-28688
| Versions      | 2.3, master

Updates the tutorial to use new Page Builder.

Because new Content Scheduler block does not have an overflow option, the scenario is changed to use only one block of this type.

To do:
- [x] Update code in the repository

### CODE REVIEW

File: docs/tutorials/enterprise_beginner/3_use_existing_blocks.md [hunk 1]
```diff
@@ -238,153 +215,37 @@ The Block is already operational, but first update the stylesheet. Add the follo
...
-![The Page after adding a List Schedule Block](img/enterprise_tut_after_list_schedule_block.png "The Page after adding a List Schedule Block")
-
-### Set up overflow
-
-Now you will make use of the overflow functionality.
-
-In the settings of the Featured Articles block turn on overflow and select the Other Articles block as its overflow target.
+Now open the Timeline at the top of the screen.
+You can move the slider to different times and preview what the Content Scheduler block will look like at different hours.
+Content will be hidden if you moved to a point before it airs.
```
Comment (MagdalenaZuba) [inline docs/tutorials/enterprise_beginner/3_use_existing_blocks.md]: Content will be shown when you move slider to a point when it airs. - preview shows current view and you have to move the slider to see the article

File: docs/tutorials/enterprise_beginner/3_use_existing_blocks.md [hunk 2]
```diff
@@ -238,153 +215,37 @@ The Block is already operational, but first update the stylesheet. Add the follo
...
-You may know this behavior from many websites.
-When a new article appears in the first Schedule block, the last article currently in it will be 'pushed off'
-and will land in the block designated as the overflow block – this means in the list of articles below.
-This way the most current articles are shown at the top, while older articles are still easily accessible from the front page.
+!!! tip
 
-You can try it out now. Add one more Article to the Featured Articles block.
-You will see a warning that some content will be pushed out.
-When you confirm, the pushed out Article will move to the top of the Other Articles block.
+    At this point you have configured the Content Scheduler block to work well with Articles only.
```
Comment (MagdalenaZuba) [inline docs/tutorials/enterprise_beginner/3_use_existing_blocks.md]: to work with Articles only.

File: docs/tutorials/enterprise_beginner/3_use_existing_blocks.md [hunk 3]
```diff
@@ -55,14 +38,14 @@ First, create an override template for the Content List block: `app/Resources/vi
...
-In `app/config/layouts.yml` add the following code at the end of the file, under the `ez_systems_landing_page_field_type` key on the same level as `layouts`:
+In `app/config/layouts.yml` add the following code at the end of the file, under the `ezplatform_page_fieldtype` key on the same level as `layouts`:
 
 ``` yaml
 blocks:
     contentlist:
         views:
-            contentList:
-                template: 'blocks/contentlist.html.twig'
+            contentlist:
```
Comment (ciastektk) [inline docs/tutorials/enterprise_beginner/3_use_existing_blocks.md]: To keep with our new structure we need to change this key to default

File: docs/tutorials/enterprise_beginner/3_use_existing_blocks.md [hunk 4]
```diff
@@ -126,36 +107,35 @@ The block is displayed using the new template.
...
-The next block is the Schedule block that will air articles at predetermined times.
-You will use two different Schedule blocks with different templates and set up overflow between them.
+The next block is the Content Scheduler block that will air articles at predetermined times.
 
 First, add a configuration that points to the layout. Go to `app/config/layouts.yml` again and add the following code under `blocks` on the same level as the `contentlist` key:
 
 ``` yaml
 schedule:
     views:
         schedule_featured:
```
Comment (ciastektk) [inline docs/tutorials/enterprise_beginner/3_use_existing_blocks.md]: And this one to featured

**Pull request merged**